services/news.js

```diff
@@ -14,6 +14,24 @@ const restrictedFields = [];
 const defaultFields    = ['*'];
 const queryBuilder     = new QueryBuilder(News, restrictedFields, defaultFields);
 
+const checkSlugExist = async (doc) => {
+    const arg = {$or: []};
+    if (doc.translation) {
+        for (const lang of Object.entries(doc.translation)) {
```

Comment:
instead of :
```js
for (const lang of Object.entries(doc.translation)) {
  if (doc.translation[lang[0]]) {
    arg.$or.push({[`translation.${lang[0]}.slug`]: doc.translation[lang[0]].slug});
  }
}
```
you could write it like that :
```js
// descructuring resulting array from Object.entries
for (const [lang] of Object.entries(doc.translation)) {
  if (doc.translation[lang]) {
    arg.$or.push({[`translation.${lang}.slug`]: doc.translation[lang].slug});
  }
}
```